cnf-certification-test/accesscontrol/resources/resources_test.go

```diff
@@ -83,3 +83,122 @@ func TestHasRequestsAndLimitsSet(t *testing.T) {
 		assert.Equal(t, tc.expectedResult, HasRequestsAndLimitsSet(tc.testContainer))
 	}
 }
+
+//nolint:funlen
+func TestHasExclusiveCPUsAssigned(t *testing.T) {
+	testCases := []struct {
+		testContainer  *provider.Container
+		expectedResult bool
+	}{
+		{ // Test Case #1 - Exclusive CPU pool selected
+			testContainer: &provider.Container{
+				Container: &corev1.Container{
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							"cpu":    resource.MustParse("2"),
+							"memory": resource.MustParse("512Mi"),
```

Comment:
I think some constants are defined but not used. Could you verify this please?

Reply:
Done.